README.md

```diff
@@ -1,2 +1,68 @@
-# module1-capstone
- HTML + CSS + JS project to implement concert or conference web site.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Module 1 - Capstone Project
+
+HTML + CSS + JS project to implement International Conference on Open-Water Swimming conference web site.
+
+The website it's Mobile-first designed and have breakpoints for 768px and less, between 768px and 1024px, and 1024px and more screen size devices.
+
+The functionality available is the Landing Page and the About Page.
+
+For the landing page, the sections to display the program topics and the relevant speakers are built dynamically by reading the info from data files in JavaScript.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Bootstrap CSS Framework for styling.
+
+## Live Demo
+
+https://lmunozm1702.github.io/module1-capstone/
+
+## Getting Started
+
+### Prerequisites
+
+### Setup
+
+### Install
```

Comment:
- [ ] I think it would be better to add some instructions or steps on how to install and run your project in `Getting Started` section, such as `git clone <project url>`, `cd <project name>`, ... 💯 

- [ ] Kindly, Remove the rest of empty sections in your `README` 👍

> We should always be sure our repositories are documented professionally, please fix broken links on the readme file.